FullStoryMiddlewareExample/AppDelegate.m

```diff
@@ -16,7 +18,35 @@ @implementation AppDelegate
 
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    // Override point for customization after application launch.
+    
+    // set certain events to be the only ones allowed to forward to FullStory
+    NSArray *allowList = @[@"Product Viewed", @"Order Completed", @"Completed Checkout Step"];
+    FullStoryMiddleware *fsm = [[FullStoryMiddleware alloc] initWithAllowlistEvents: allowList];
+    
+    // enable to insert FS session URL to Segment event properties and contexts
+    // default to true
+    fsm.enableFSSessionURLInEvents = true;
+    // when calling Segment group, send group traits as userVars
+    // default to false
+    fsm.enableGroupTraitsAsUserVars = true;
+    // when calling Segment screen, sent the screen event as custom events to FS
+    // default to false
+    fsm.enableSendScreenAsEvents = true;
+    // allow all track events as FS custom events
+    // alternatively allow list events that you would like to track
+    // note: enabling this will cause the middleware to ignore the event allowlist
+    // default to false
+    fsm.allowlistAllTrackEvents = true;
```

Comment:
These comments would be easier to read if they're separated out a bit more. Capitalization and punctuation also help.

I also wasn't sure what you meant by "default to X", since that sounds like you're setting a default below the comment, but the value is usually different. So I guessed you meant the initial default value before this point, in which case "default is X" sounds clearer.
```suggestion
    // Enable to insert FS session URL to Segment event properties and contexts.
    //
    // Default is true.
    fsm.enableFSSessionURLInEvents = true;
    
    // When calling Segment group, send group traits as userVars.
    //
    // Default is false.
    fsm.enableGroupTraitsAsUserVars = true;
    
    // When calling Segment screen, send the screen event as custom events to FS.
    //
    // Default is false.
    fsm.enableSendScreenAsEvents = true;
    
    // Allow all track events as FS custom events.
    // Alternatively, allow list events that you would like to track.
    //
    // Note: enabling this will cause the middleware to ignore the event allowlist.
    //
    // Default is false.
    fsm.allowlistAllTrackEvents = true;
```